### PR TITLE
Not unauthenticated, but those that cannot be processed

### LINKED ISSUE #4711: Ben Kaduk's Recovery Comment 4
@kaduk said

> Section 6.2.3
> 
> > Endpoints
> > that do not cease retransmitting packets in response to
> > unauthenticated data risk creating an infinite exchange of packets.
> 
> We may want to be more precise about what "unauthenticated data" means
> here, since in -transport we say that even Initial packets are
> "authenticated" in some sense.

### PR DESCRIPTION
**Pull request by janaiyengar**
Closes #4711.

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -683,12 +683,12 @@ client receives Handshake or 1-RTT packets prior to obtaining Handshake keys,
...
 To speed up handshake completion under these conditions, an endpoint MAY, for a
-limited number of occasions per each connection, send a packet containing
+limited number of times per connection, send a packet containing
 unacknowledged CRYPTO data earlier than the PTO expiry, subject to the address
 validation limits in Section 8.1 of {{QUIC-TRANSPORT}}. Doing so at most once
 for each connection is adequate to quickly recover from a single packet loss.
-Endpoints that do not cease retransmitting packets in response to
-unauthenticated data risk creating an infinite exchange of packets.
+An endpoint that always retransmits packets in response to receiving packets
+that it cannot process risks creating an infinite exchange of packets.
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md:691]:
```suggestion
An endpoint that always retransmits packets in response to receiving
unprocessable packets risks creating an infinite exchange of packets.
```

**Review by ianswett — APPROVED** [review]
LG, one suggestion

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-recovery.md]: I don't like this change.

Comment (janaiyengar) [reply draft-ietf-quic-recovery.md]: Yeah, I think that this changes what an endpoint cannot do right now to a property of the packets. I'm reverting this change.

**Pull request merged**